docs/agglayer/get-started/bridge-assets.md

```diff
@@ -139,25 +151,27 @@ Before proceeding with claiming the asset, you need to verify its current status
 
 Using Postman, import the API endpoint to check the status of your transaction. This query will return details such as the token bridged, the amount, and the current state of the transaction. Remember to get your API Key via this [link](https://polygontechnology.notion.site/api-gateway-service-documentation).
```

Comment:
Please use this link -> https://polygontechnology.notion.site/Api-Gateway-Service-Documentation-13b80500116a80439ee3c34a850c416b